### ISSUE TITLE
Verify if past data will overflow

### ISSUE THREAD

**Issue by MedadRufus** [ImperialSpaceSociety/LoRaMac-node#291]
Verify this struct:
```c
typedef struct
{
  uint32_t minutes_since_epoch; // minutes since Epoch
  uint16_t latitude_encoded;        // Latitude * 65536
  uint16_t longitude_encoded;       // Longitude * 65536
  uint16_t altitude_encoded;     // Altitude in meters / 256

}time_pos_fix_t;
```

Will it be an issue if it goes over 170 degrees longitude?

**Comment by MedadRufus** [comment]
create an encode and decode function that can be unittested. Then test the full range of expected values.

**Comment by MedadRufus** [comment]
fixed in #294
